Algorithms/arrays/leftShift/README.md

```diff
@@ -0,0 +1,53 @@
+# Algorithms
+
+## METHOD 1 (NAIVE) <br>
+Time = O(n*d) 
```

Comment:
For complexity formulas, the formula should be written in a way, that it is rendered properly, rather than plain text. 
(You could check [this](https://github.com/abxhr/Open-DSA/blob/main/Algorithms/Searching/README.md), to get a better idea) 

**eg**: 
```
Worst Case: ![formula](https://render.githubusercontent.com/render/math?math=O(n))
Average Case: ![formula](https://render.githubusercontent.com/render/math?math=O(n))
Best Case: ![formula](https://render.githubusercontent.com/render/math?math=O(1))
```
is rendered as:

Worst Case: ![formula](https://render.githubusercontent.com/render/math?math=O(n))
Average Case: ![formula](https://render.githubusercontent.com/render/math?math=O(n))
Best Case: ![formula](https://render.githubusercontent.com/render/math?math=O(1))

**Implement this for all math equations in this `README.md` file, and also the `README.md` file for `rightShift`.**